packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -103,6 +103,10 @@ class CupertinoCheckbox extends StatefulWidget {
     this.tristate = false,
     required this.onChanged,
     this.activeColor,
+    @Deprecated(
+      'Use fillColor instead. '
```

Comment:
Not sure if this is enough information to help someone migrating grasp what they need to do in an instant? Or what do you think @MitchellGoodwin?

Reply:
I think so too. Maybe another line that mentions fillColor controls the color in all states, including when not selected.